### DISCUSSION TITLE
Replace className with cn

### DISCUSSION THREAD

**Discussion by AndresdoSantos** [nativewind/nativewind#462] [Q&A]
Hi everyone, I have a project here that apparently is going to be very large and I want to reduce the writing on it.

So I created a typing to remove the className and add the cn.

```
export type TRemoveClassName<T> = Omit<T, 'className'> & {
  cn?: string
}
```

But VSCode intellisense doesn't understand "cn" like I understand className.

Is there any way to make VSCode understand "cn" and make it work the same as className?

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
